crates/topos-tce-storage/src/validator/tables.rs

```diff
@@ -29,13 +34,17 @@ impl ValidatorPendingTables {
 
         let cfs = vec![
             ColumnFamilyDescriptor::new("pending_pool", default_options()),
+            ColumnFamilyDescriptor::new("pending_pool_index", default_options()),
             ColumnFamilyDescriptor::new("precedence_pool", default_options()),
         ];
 
         let db = init_with_cfs(&path, default_options(), cfs)
             .unwrap_or_else(|_| panic!("Cannot open DB at {:?}", path));
 
         Self {
+            // TODO: Fetch it from the storage
+            next_pending_id: AtomicU64::new(0),
```

Comment:
Maybe in preparation for this TODO we should have `ValidatorPendingTables::next_id()` already?

Reply:
We'll introduce a new method if needed when working on the pending subject. Maybe the `next_pending_id` will be removed, I don't know yet.